executor/aggfuncs/aggfuncs.go

```diff
@@ -38,6 +38,9 @@ var (
 	// All the AggFunc implementations for "MAX" are listed here.
 	// All the AggFunc implementations for "MIN" are listed here.
 	// All the AggFunc implementations for "GROUP_CONCAT" are listed here.
+	_ AggFunc = (*groupConcat4DistinctString)(nil)
+	_ AggFunc = (*groupConcat4String)(nil)
```

Comment:
`group_concat` always executes on the string inputs, I think we can remove the `4String` suffix. How about `groupConcat` and `groupConcatDistinct`?